src/app/components/Curation/index.tsx

```diff
@@ -143,31 +141,31 @@ export default ({
     case SIMPLE_CURATION_GRID:
     case HIERARCHICAL_CURATION_GRID:
     default:
-      return curationLength > 1 &&
-        summaries.length > 0 &&
-        (title || isFirstCuration) ? (
-        <section aria-labelledby={id} role="region">
-          {isFirstCuration ? (
-            <VisuallyHiddenText id={id} as="h2">
-              {curationSubheading}
-            </VisuallyHiddenText>
-          ) : (
-            <Subheading id={id} link={link}>
-              {curationSubheading}
-            </Subheading>
-          )}
+      if (summaries.length > 0) {
+        return curationLength > 1 && (title || isFirstCuration) ? (
+          <section aria-labelledby={id} role="region">
+            {isFirstCuration ? (
+              <VisuallyHiddenText id={id} as="h2">
+                {curationSubheading}
+              </VisuallyHiddenText>
+            ) : (
+              <Subheading id={id} link={link}>
+                {curationSubheading}
+              </Subheading>
+            )}
+            <GridComponent
+              summaries={summaries}
+              headingLevel={3} // multiple curations, don't we want the promos to all be h3?
```

Comment:
Let's add a comment to explain what's going on

```suggestion
              headingLevel={3} // if there are multiple curations, each curation's heading will be h2 and the promos within will be h3
```